### ISSUE TITLE
Request for WebCode2M Test Dataset

### ISSUE THREAD

**Issue by bearnichi** [CGCL-codes/naturalcc#50]
Hiii！I am a beginner trying to reproduce UICopilot (arXiv:2505.09904) and encounter difficulties in getting the official WebCode2M test set :
1. The WebCode2M dataset on Hugging Face only provides the training set .
2. The UICopilot code uses an absolute server path for the test set (e.g., `/data02/projects/vision2ui/datasets/{test_data_name}_benchmark`), and I cannot find a download link or "test/" folder in the repository.
Could you please share the download link for the WebCode2M test set or provide instructions on how to obtain it?
I think your research is extremely valuable！Thank you very much for your help!

**Comment by bearnichi** [comment]
find it：https://huggingface.co/datasets/xcodemind/webcode2m_test
